AutoRest/Generators/CSharp/CSharp/Templates/ModelTemplate.cshtml

```diff
@@ -2,6 +2,7 @@
 @using Microsoft.Rest.Generator.ClientModel
 @using Microsoft.Rest.Generator.Utilities
 @using Microsoft.Rest.Generator
+@using Microsoft.Rest.Generator.CSharp
```

Comment:
Is this using necessary? The ShouldValidateChain() method is called in this template and is also defined in ClientModelExtensions.cs, but we didn't need the using before.